IPython/core/tests/test_interactiveshell.py

```diff
@@ -171,9 +171,9 @@ def test_can_pickle(self):
                      "    def __init__(self,x=[]):\n"
                      "        list.__init__(self,x)"))
         ip.run_cell("w=Mylist([1,2,3])")
-        
```

Comment:
Yeah, sorry for the whitespace. I'm using vim and vim gonna do vim stuff.I will clean the PR.